docs/SRS-Volere/SRS.tex

```diff
@@ -811,7 +811,12 @@ \section{Waiting Room}
 \lips
 
 \section{Ideas for Solution}
-\lips
+\begin{enumerate}
```

Comment:
There's no mention of the documentation management aspect here. Is that intentional?

Reply:
I can't remember if we had any ideas for document management which isn't a FR/NFR. Imk and ill add it in.

Reply:
Should be fine then, just wondering